pkg/services/policer/policer.go

```diff
@@ -65,6 +65,14 @@ type Option func(*cfg)
 // the redundant local copy of the object.
 type RedundantCopyCallback func(oid.Address)
 
+// Network provides information about the NeoFS network to Policer for work.
+type Network interface {
```

Comment:
Not sure about naming here and below, but we're already using `NodeState` and `NodeInfo`.

Reply:
Maybe `LocalConfig`, because that's what we have in `shared`.

Reply:
This is a global network state, for example, is node in the network map or not.